bin/templates/project/__CLI__.xcodeproj/project.pbxproj

```diff
@@ -374,7 +374,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cordova/lib/copy-www-build-step.sh";
+			shellScript = "node cordova/lib/copy-www-build-step.js";
```

Comment:
+1 The cli dependent on node, so it makes sense to require it in the path.
Does this affect users who are not using the cli? Probably still not a big deal either way.